strapi/.env.example

```diff
@@ -1,5 +1,8 @@
+# To start developing, copy this into .env.local
+
 HOST=0.0.0.0
 PORT=1337
+DATABASE_HOST=127.0.0.1
```

Comment:
If we change `localhost` to `127.0.0.1` here https://github.com/bratislava/marianum.sk/blob/a323f9ea5b850cf2ab373eed8f51779355c6de57/strapi/config/database.ts#L5, we probably don't need DATABASE_HOST in `strapi/.env.example`